### PR TITLE
RFC: opengl-es-cts: add Khronos OpenGL ES CTS

### PR DESCRIPTION
**Pull request by lumag**
Signed-off-by: Dmitry Baryshkov <dmitry.baryshkov@linaro.org>

### CODE REVIEW

**Comment by lumag** [comment]
It is still an RFC, I'll take a look onto the Vulkan CTS later.

File: recipes-graphics/vk-gl-cts/opengl-es-cts_3.2.6.1.bb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+SRCREV_spirv-headers = "e4322e3be589e1ddd44afb20ea842a977c1319b8"
+SRCREV_spirv-tools = "1eb89172a82b436d8037e8a8c29c80f7e1f7df74"
+
+S = "${WORKDIR}/git"
+
+inherit pkgconfig cmake
+
+DEPENDS += "libpng zlib virtual/libgles2 virtual/egl"
+
+EXTRA_OECMAKE_append = "-DDEQP_TARGET=surfaceless"
```
Comment (ndechesne) [inline recipes-graphics/vk-gl-cts/opengl-es-cts_3.2.6.1.bb:28]: you need a leading whitespace here.

**Comment by lumag** [comment]
@alimon updated to include separate vulkan cts.

**Comment by ndechesne** [comment]
@lumag I suppose we can abandon this now since it's merged in meta-oe?

**Comment by lumag** [comment]
Yes, closing.

**Pull request closed without merging**